io/net/bridge.py

```diff
@@ -80,28 +67,22 @@ def test(self):
 
         self.check_failure('ip link set %s master br0' % self.host_interface)
         self.check_failure('ip addr flush dev %s' % self.host_interface)
-        self.check_failure('ip addr add %s broadcast %s dev br0' %
-                           (self.cidr, self.broadcast))
+        self.check_failure('ip addr add %s/%s dev br0' %
+                           (self.ipaddr, self.netmask))
```

Comment:
Why is this needed if we are already doing `configure_network.set_ip()`

Reply:
In configure_network.set_ip() , we are only configuring the host interface. 
Assigning IP is done with the above method.

Reply:
@vaishnavibhat Can you explain a bit more on the difference please ?

Reply:
in the yaml file, we enter the interface name , IP and netmask of the interface that is a part of the bridge interface. configure_network.set_ip() with create a cfg file for this, get the interface up and check if it is working fine. 
The later part of the code is used to create bridge, configure it and do a ping check and do not need a cfg file. 
configure_network utility is available only for configuring normal interface and not for other devices.

Reply:
please enable it for bridge interfaces @vaishnavibhat

Reply:
Using the  configure_network.set_ip() requires some changes in the  configure_network.py utility. 
Working on it.